src/test/resources/data/FeedbackRankQuestionUiTest.json

```diff
@@ -1109,7 +1109,7 @@
       "responseMetaData": {
         "value": "{\"answer\":2,\"questionType\":\"RANK_RECIPIENTS\"}"
       },
-      "giverSection": "None",
+      "giverSection": "Section 1",
```

Comment:
Self feedback rank recipient questions are no longer valid (#8218), but this is not enforced in the backend, we need to create a corresponding issue for it.

Reply:
@mmoanis You can either go ahead and remove this question from test data or see if it is possible to modify the question in an easy way